### ISSUE TITLE
Understanding Success Criterion 1.4.11: Non-text Contrast - Border & Box Shadow

### ISSUE THREAD

**Issue by LaurenceRLewis** [w3c/wcag#3140]
Does the following pattern meet the requirements for 1.4.11: Non-text Contrast?
_This is only related to a checked checkbox that receives keyboard focus._

When the checkbox is checked, the background is a dark blue and the checked indicator is a white tick. this passes all WCAG requirements.
However, when the checked checkbox receives keyboard focus, a border is added, the border is the same colour as the check background. Additionally there is a box-shadow of light blue.

The light blue box shadow does not meet the 3.1:1 contrast ratio against the white background, but the dark border does.

**Which colour is used to measure the contrast, the dark blue border or the light blue box shadow?**

**Image examples:**

Checked unfocused:
![image](https://user-images.githubusercontent.com/4565139/230238413-5ef14fc6-7e60-4f0f-9b80-8bb065112546.png)

Checked focused:
![image](https://user-images.githubusercontent.com/4565139/230238384-2d374e2e-a166-4331-8b22-901a1e041ce2.png)

Unchecked focused showing the border and box shadow that is added to the checked version: 
![image](https://user-images.githubusercontent.com/4565139/230238345-086b3c20-5ed8-4710-a5f9-8a911a412d33.png)

**Comment by LaurenceRLewis** [comment]
There is a paragraph in the Understanding 1.4.11 Non-text Contrast (Level AA)
"If components use several colors, any color which does not interfere with identifying the component can be ignored for the purpose of measuring contrast ratio. For example, a 3D drop-shadow on an input, or a dark border line between contrasting backgrounds is considered to be subsumed into the color closest in brightness (perceived luminance)."

However, I don't know if the example I provided meets this condition.

**Comment by LaurenceRLewis** [comment]
The light blue gets subsumed into the color closest in brightness the dark blue border.
